week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -38,7 +58,19 @@ Write a function that:
 */
 
 function formatPercentage(arr) {
-  
+  let myArray = arr;
+  let newArray = [];
```

Comment:
Just a small comment about naming that might help in the future. It is always good to use variable names that convey information about what they contain. Could you think of a way to change the name to express that this array contains the numbers rounded to 2 decimal points?

Reply:
toDecimalsArr